content/millennium/r4/financial/general/charge-item.md

```diff
@@ -83,19 +83,22 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?       | Type          | Description
------------------------|-----------------|---------------|-------------------------------------------------------
- `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
- `context`             | This or _id     | [`token`]     | Encounter associated with event.
- `-status`             | See notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
- [`_count`]            | No              | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                  | Type          | Description
+-----------------------|----------------------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context or account | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id or account     | [`reference`] | Encounter associated with event.
```

Comment:
Not related to this change, but can we go ahead and change this Description to "Encounter associated with the ChargeItem."

Reply:
Addressed in 2e4df3f24bdeb9538a6e6875435ea5f695bd59b5

Reply:
Thanks for making that change.